client/js/lounge.js

```diff
@@ -300,10 +300,6 @@ $(function() {
 			text.html(templates.actions[type](data.msg));
 		}
 
-		if (text.find("i").size() === 1) {
-			text = text.find("i");
```

Comment:
Shout's parser had some unnecessary `<i>` wrapping which we removed. And this code isn't used anymore.